### PR TITLE
[#1] Update readme: Add badges

### LINKED ISSUE #1: Add Repo CI / Coverage / Vulnerabilities badge
I've never done this, but here are a few resources:

- https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge
- https://github.com/dwyl/repo-badges

### PR DESCRIPTION
**Pull request by newswim**
Closes #1 

### Overview

I used mostly used https://shields.io for the badge generation. The code coverage was a little more involved, so I followed a workflow that I found here: https://github.com/marketplace/actions/jest-badges-generation-action. I'm not totally sure if it's configured correctly, and I'm also not sure exactly what `branches: main` means here..

```yml
# build.yml
[...]
      - run: yarn run eslint
      - run: yarn test:ci
      - name: Generating coverage badges
        uses: jpb06/jest-badges-action@latest
          with:
            branches: main
```

### Preview:

image

Note: that the CI step will need to run before the code coverage badge is generated. And it's highly likely that I messed something up :)

### CODE REVIEW

File: LICENSE
Comment (skeate) [inline LICENSE:3]: TIL Jacob is an immortal

File: .github/workflows/build.yml
Comment (skeate) [inline .github/workflows/build.yml:55]: 🤔  This will create new commits every time, right?

There are other methods which involve setting up the tests to report coverage to a third party (e.g. [Coveralls](https://coveralls.io/)[^1]) and then use a badge they generate. IMO that's preferable to something creating a bunch of extraneous commits (especially since you can then click the badge and see the coverage report!)

[^1]: It's been a long time since I've set up a coverage badge for a repo, so while I remember this one, I'm sure others have come along that might be better. This is not a "we should use Coveralls" post

File: LICENSE
Comment (jacob-alford) [reply LICENSE:3]: 'twas a great year!

File: .github/workflows/build.yml
Comment (jacob-alford) [reply .github/workflows/build.yml:55]: Is there a reason that we wouldn't want to commit for this workflow?  I don't like the idea of robot commits, but I think it's a free option, and reassuring that such a new package has such high test coverage

**Review by jacob-alford — APPROVED** [review]
Thanks for getting the badges set up!  I'm not completely sure about the coverage badge, it seems like it might add more complexity than it's worth.  But I say we try it!  And if it isn't what we want, we can always take it out.

File: .github/workflows/build.yml
Comment (skeate) [reply .github/workflows/build.yml:55]: I just mean it's an _additional_ commit after any time there's a merge to main. Not my project so I'll leave it up to you, but I like a third party service, since (a) they're usually free for FOSS, (b) they host the actual coverage results, (c) they can show you the coverage history, and (d) they don't require an additional commit to publish a badge

Comment (jacob-alford) [reply .github/workflows/build.yml:55]: That seems like it's worth it, I say we do it.  It'd be nice to have the coverage history

Comment (newswim) [reply .github/workflows/build.yml:55]: Hm, yeah I think it's better to have a clean git history. Where "clean" means mostly atomic commits and little-to-no robot commits. I'll look into using `coveralls` or some other workflow.

**Review by jacob-alford — COMMENTED** [review]
I went ahead and signed up for Coveralis, so once that's all set up I'll give it a re-approval

File: README.md
Comment (jacob-alford) [inline README.md:5]: 🎖️  this is freakin' beautiful 🥹

**Review by jacob-alford — APPROVED** [review]
🙌🏻🙌🏻🙌🏻

**Pull request merged**
